documentation/asciidoc/accessories/iqaudio/configuration.adoc

```diff
@@ -18,66 +18,59 @@ Find the `dtparam=audio=on` line in the file and comment it out by placing a # s
+All the necessary mounting hardware including spacers, screws and connectors is provided. The PCB spacers should be screwed, finger-tight only, to the Raspberry Pi before adding the audio board. The remaining screws should then be screwed into the spacers from above.
 
-=== Codec Zero Configuration
+=== Codec Zero configuration
 
 The Raspberry Pi Codec Zero board uses the Dialog Semiconductor DA7212 codec. This allows
-the recording of audio from the built in MEMS microphone, from stereo Phono sockets (AUX
-IN) or 2 x mono external Electret microphones. Playback is through stereo Phono sockets (AUX OUT)
+the recording of audio from the built-in MEMS microphone, from stereo phono sockets (AUX
+IN) or two mono external electret microphones. Playback is through stereo phono sockets (AUX OUT)
 or a mono speaker connector.
 
-Each input and output device has its own “mixer” allowing the audio levels and volume to be adjusted
-independently. Within the codec itself other mixers and switches exist to allow the output to be mixed to a single mono channel
-for single speaker output. Signals may also be inverted and there is a 5 band Equaliser to adjust
-certain frequency bands. These settings can be controlled interactively using Alsamixer (a graphical mixer program used in Terminal) or programmatically.
+Each input and output device has its own mixer, allowing the audio levels and volume to be adjusted
+independently. Within the codec itself, other mixers and switches exist to allow the output to be mixed to a single mono channel for single-speaker output. Signals may also be inverted; there is a five-band equaliser to adjust certain frequency bands. These settings can be controlled interactively, using AlsaMixer, or programmatically.
 
-It is important to note that the AUX IN and AUX OUT are both 1V RMS. It may be necessary to adjust
-the AUX IN’s mixer to ensure the input signal doesn’t saturate the ADC (Analogue to Digital
-Convertors). Similarly, the output mixers can be to be adjusted to get the best possible output.
+Both the AUX IN and AUX OUT are 1V RMS. It may be necessary to adjust
+the AUX IN’s mixer to ensure that the input signal doesn’t saturate the ADC. Similarly, the output mixers can be to be adjusted to get the best possible output.
 
-There is a set of preconfigured scripts (loadable ALSA settings) available on GitHub. https://github.com/iqaudio/Pi-Codec.
-
-These cover several use cases such as:
+Preconfigured scripts (loadable ALSA settings) are available on GitHub. https://github.com/iqaudio/Pi-Codec, offering:
  
 * Mono MEMS mic recording, mono speaker playback
 * Mono MEMS mic recording, mono AUX OUT playback
 * Stereo AUX IN recording, stereo AUX OUT playback
 * Stereo MIC1/MIC2 recording, stereo AUX OUT playback
```

Comment:
If we haven't already, I guess we ought to check that all of these combinations **do** actually work?

Reply:
Off you go then! (Not now; you've got other things to be doing, but if you wouldn't mind assigning yourself a task in Asana for some time in the bold future, that'd be great.)